### PR TITLE
feat: Translations in es_MX

### PR DESCRIPTION
**Pull request by AndreyPootMay**
I add the translation for confirmShowSolution, nextLevel, sandbox, IntlHelperBarView, étc. Various of the levels.

I hope not to affect the work of the other contributors.

If I could request only one thing, it would be for the pull-request to be added the tag "hacktoberfest-accepted" to the PR. As well as the "hacktoberfest" topic to the repository, although I hope not to bother.

### CODE REVIEW

File: src/js/dialogs/confirmShowSolution.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ exports.dialog = {
     type: 'ModalAlert',
     options: {
       markdowns: [
-        '## ¿Estás seguro de que quiere ver la solución?',
+        '## ¿Estás seguro de que quieres ver la solución?',
```
Comment (waldyrious) [inline src/js/dialogs/confirmShowSolution.js:56]: 👍 I was going to suggest this change in #745 but I assumed it was a local variation. 😄

File: src/js/dialogs/sandbox.js [hunk 2]
```diff
@@ -78,6 +78,62 @@ exports.dialog = {
...
+  'es_MX': [{
+    type: 'ModalAlert',
+    options: {
+      markdowns: [
+        '## ¡Bienvenid@ a Learn Git Branching!',
+        '',
+        'Esta aplicación está diseñada para ayudar a los principantes ',
+        'a manejar los poderosos conceptos que hay detrás del trabajo ',
+        'con ramas (branches) en Git. Esperamos que disfrutes la aplicación ',
+        'y tal vez incluso ¡que aprendas algo! ',
```
Comment (waldyrious) [inline src/js/dialogs/sandbox.js:90]: Is the space at the end of the line necessary?

File: src/js/dialogs/sandbox.js [hunk 3]
```diff
@@ -78,6 +78,62 @@ exports.dialog = {
...
+        '[https://pcottle.github.io/learnGitBranching/?NODEMO](?NODEMO)'
+      ]
+    }
+  }, {
+    type: 'ModalAlert',
+    options: {
+      markdowns: [
+        '## Comandos de Git',
+        '',
+        'Tienes una gran variedad de comandos de git en este sandbox. He aquí una lista de los incluidos: ',
```
Comment (waldyrious) [inline src/js/dialogs/sandbox.js:109]: Elsewhere "Git" is capitalized. Shouldn't it be here as well?

Comment (waldyrious) [reply src/js/dialogs/sandbox.js:109]: Also, extra space at the end of the string.

File: src/js/dialogs/sandbox.js (hunk 2 above)
Comment (AndreyPootMay) [reply src/js/dialogs/sandbox.js:90]: Corrected

File: src/js/dialogs/sandbox.js (hunk 3 above)
Comment (AndreyPootMay) [reply src/js/dialogs/sandbox.js:109]: Corrected

**Comment by pcottle** [comment]
Looks great! Thanks you two for iterating on it as well 🙌🏻

**Comment by pcottle** [comment]
(Merged this last night but just pushing the site with the updates now)

**Pull request merged**